Gems/EMotionFX/Code/Source/Editor/Plugins/ColliderWidgets/HitDetectionJointWidget.h

```diff
@@ -23,30 +23,35 @@ namespace EMotionFX
     class AddColliderButton;
     class ColliderContainerWidget;
 
-    class HitDetectionJointWidget
-        : public SkeletonModelJointWidget
+    class HitDetectionJointWidget : public SkeletonModelJointWidget
     {
-        Q_OBJECT //AUTOMOC
+        Q_OBJECT // AUTOMOC
 
-    public:
-        HitDetectionJointWidget(QWidget* parent = nullptr);
+            public : HitDetectionJointWidget(QWidget* parent = nullptr);
```

Comment:
can you fix the indent here? I know that the Q_OBJECT Marco always mess up in here if you leave it automatic.